assets/templates/dining_lists/dining_slot_diners.html

```diff
@@ -6,176 +6,163 @@
-                                                            {% if entry.has_shopped %} checked {% endif %}>
-                                                    Shop
-                                                </label>
-                                                <label class="btn btn-outline-primary col-3
-                                                        {% if entry.has_cooked %} active {% endif %}">
-                                                    <input type="checkbox" autocomplete="off"
-                                                           id="{{ entry.id }}:cook" name="{{ entry.id }}:has_cooked"
-                                                            {% if entry.has_cooked %} checked {% endif %}>
-                                                    Cook
-                                                </label>
-                                                <label class="btn btn-outline-primary col-3
-                                                        {% if entry.has_cleaned %} active {% endif %}">
-                                                    <input type="checkbox" autocomplete="off"
-                                                           id="{{ entry.id }}:clean" name="{{ entry.id }}:has_cleaned"
-                                                            {% if entry.has_cleaned %} checked {% endif %}}>
-                                                    Clean
-                                                </label>
-                                            {% else %}
-                                                <div class="col-3">
-                                                    {% if entry.has_shopped %} Shop {% endif %}
-                                                </div>
-                                                <div class="col-3">
-                                                    {% if entry.has_cooked %} Cook {% endif %}
-                                                </div>
-                                                <div class="col-3">
-                                                    {% if entry.has_cleaned %} Clean {% endif %}
-                                                </div>
-                                            {% endif %}
-                                        {% else %}
-                                            <div class="col-9"></div>
+                <tbody>
+                {# Table rows #}
+                {% for entry in entries %}
+                    {% with entry=entry.get_subclass %}
+                        <tr>
+                            <td>
+                                <div class="d-flex justify-content-between align-items-center">
+                                    <span>
+                                        {{ entry.get_name }}
+                                        {% if entry.is_external or entry.created_by != entry.user %}
```

Comment:
I'm not a fan of displaying the user that added the account; it serves no purpose and can clutter the list (especially in mobile format). The reason it is displayed on external users is that the claimer can trace where the new person originates from. However, for already existing accounts this is less of an issue. Thus I'd like to see the added by no longer displayed on internal entries.

Reply:
I'll change it